cardano_node_tests/utils/clusterlib_utils.py

```diff
@@ -1615,3 +1615,36 @@ def get_just_lovelace_utxos(
     return cl_txtools._get_usable_utxos(
         address_utxos=address_utxos, coins={clusterlib.DEFAULT_COIN}
     )
+
+
+def check_ratify_state(
```

Comment:
The function is doing two unrelated things:
1. checking that all the expected fields are in the query output
2. checking that the expected action was ratified

Let's split it into two things:
1. a simple test in `test_conway.py` that checks the fields
2. a check function in `governance_utils.py` that will be very similar to the existing `lookup_ratified_actions` function.
3. 
FYI you need bot action txid and ix to match the action, as there could be multiple governance actions in single transaction.